cfc-testing/scripts/compliance-report-generator.py

```
@@ -0,0 +1,1389 @@
+
+            print(f"   ✅ Parsed {total_testcases} testcases: {updated_count} updated, {created_count} created, {skipped_count} skipped (non-CSV)")
+
+            # Diagnostic: Show sample of parsed data
+            if results_by_config:
+                sample_config = next(iter(results_by_config.keys()))
+                sample_duration = results_by_config[sample_config].get('duration', 0)
+                print(f"   📊 Sample: {sample_config} = {sample_duration:.2f}s")
+            else:
+                print(f"   ⚠️  WARNING: No compliance test results extracted from JUnit XML!")
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)